catroid/src/org/catrobat/catroid/content/bricks/WhenNfcBrick.java

```diff
@@ -80,6 +80,10 @@ public WhenNfcBrick(WhenNfcScript script) {
 		this.oldSelectedNfcTag = null;
 		this.nfcTag = script.getNfcTag();
 		this.whenNfcScript = script;
+
+		if (script != null && script.isCommentedOut()) {
+			setCommentedOut(true);
+		}
```

Comment:
You could copy it in the BrickBaseType. Would there be any reason, why you shouldn't copy it for every Brick ? Even if the Brick does not use it ? I think this should not be a problem, since every Brick has the Member and if it never gets used it does not matter what the value is set to anyway. But please correct me if I am overseeing something ;)